janusgraph-es/src/test/java/org/janusgraph/diskstorage/es/ElasticSearchConfigTest.java

```diff
@@ -171,19 +175,21 @@ public void testExternalMappingsViaMapping() throws Exception {
             idx.register(storeName, "date", IndexProviderTest.getMapping(idx.getFeatures(), ANALYZER_ENGLISH, ANALYZER_KEYWORD).get("date"), itx);
             fail("should fail");
         } catch (final PermanentBackendException e) {
+            log.debug(e.getMessage(), e);
         }
 
         final HttpPut newMapping = new HttpPut("janusgraph_"+storeName);
         newMapping.setEntity(new StringEntity(objectMapper.writeValueAsString(readMapping(version, "/strict_mapping.json")), Charset.forName("UTF-8")));
         executeRequest(newMapping);
 
-        // Test date property OK
+        // Test that the "date" property works well.
         idx.register(storeName, "date", IndexProviderTest.getMapping(idx.getFeatures(), ANALYZER_ENGLISH, ANALYZER_KEYWORD).get("date"), itx);
-        // Test weight property KO
+        // Test that the "weight" property throws an exception
```

Comment:
End sentence with a period, please.

Reply:
I add the period, but github did not see the change (https://github.com/JanusGraph/janusgraph/pull/813/files#diff-d14f4c03b06b5c0188718d5b34c7e098R187)

Reply:
That looks like a similar message in a different file, but I'm looking at the latest changes in this file, and it's still missing.